src/TaskList.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/macro'
+import userTask from './TaskList.json'
+import GetNextTaskButton from './GetNextTaskButton'
+import { Link, useRouteMatch } from 'react-router-dom'
+import Product from './images/product.jpg'
+import Numbers from './images/numbers.jpg'
+import Restriction from './images/restriction.jpg'
+import Sample from './images/sample.jpg'
+import Offer from './images/offer.jpg'
+import Calculator from './images/calculator.jpg'
+import Contract from './images/contract.jpg'
+import Eori from './images/eori.jpg'
+import Container from './images/container.jpg'
+import Money from './images/money.jpg'
+import Data from './images/data.jpg'
+import Pallets from './images/pallets.jpg'
+import Tax from './images/tax.jpg'
+import Deal from './images/deal.jpg'
+import Congratulations from './images/congratulations.jpg'
```

Comment:
That's a lot of imports, haha. Maybe there is a way to break this down into subcomponents? E.g. Tasklist, tasks, etc.

Perhaps there's also a less bulky way to source all the images you're importing here?

Reply:
If I would create subcomponents it will be more "hardcoding" which should be avoided. Each task would be a component and the same code has to be written over and over again. I have to import the images where the objects will be rendered, so here is the place to be based on my code structure.